### DISCUSSION TITLE
Allow formatting of dates such as dd/mm/yyyy and/or inherit from OS locale settings

### DISCUSSION THREAD

**Discussion by resherwni** [standardnotes/forum#2654] [Suggestions]
In the windows app, when viewing the Note History I noticed that the date format is in mm/dd/yyyy format. Most of the world uses the dd/mm/yyyy format. Would you please provide an option in settings to set the date format to the locale set in windows or set the date format to desired format and use this selected format throughout the app, for example in the Note History.

Windows App version 3.131.0. 

The Note History on the iphone app does use the OS date format.

![Screenshot 2023-01-11 194609](https://user-images.githubusercontent.com/71411863/211759648-8b4d4ec2-71ce-4d84-b5a5-8eb31241b09f.png)

**Comment by danielcolquitt** [comment]
+1

**Comment by zaernch** [comment]
+1

**Reply by zaernch** [reply]
Actually - I just discovered the answer here, see the screenshot:
#2342

**Comment by kmantel** [comment]
Similarly, would very much be great to manually set time formats across the application, or at least have it respect system settings (not locale defaults).
